app/containers/Vaults/saga.js

```diff
@@ -38,25 +38,32 @@ const crvAaveAddress = '0x03403154afc09Ce8e44C3B185C82C6aD5f86b9ab';
 const crvSAaveV2Address = '0xb4D1Be44BfF40ad6e506edf43156577a3f8672eC';
 const crvSAaveV1Address = '0xBacB69571323575C6a5A3b4F9EEde1DC7D31FBc1';
 
-const mapNewApiToOldApi = (oldVaults, newVaults) => {
-  const newVaultsMap = keyBy(newVaults, 'address');
-  const result = oldVaults.map((vault) => {
-    const newApy = get(newVaultsMap[vault.address], 'apy');
-    const newTvl = get(newVaultsMap[vault.address], 'tvl');
-    const isNew = get(newVaultsMap[vault.address], 'new');
-    if (!newApy) {
-      return vault;
-    }
+const mapNewApiToOldApi = (vaults) => {
+  const vaultsMap = keyBy(vaults, 'address');
+  const result = vaults.map((vault) => {
+    const newApy = get(vaultsMap[vault.address], 'apy');
+    const newTvl = get(vaultsMap[vault.address], 'tvl');
+    const isNew = get(vaultsMap[vault.address], 'new');
 
     const vaultApy = _.get(vault, 'apy', {});
+    const vaultToken = _.get(vault, 'token', {});
     const vaultApyData = _.get(vault, 'apy.data', {});
+    const displayName = vault.display_name || vault.name;
+    const tokenDisplayName = vaultToken.display_name;
+    const name = vault.display_name || vault.name;
     const mergedVault = {
```

Comment:
Override/extend data to map the new API format (snake case) to old API format (camel case)